demisto_sdk/commands/validate/tests/RM_validators_test.py

```diff
@@ -434,3 +440,105 @@ def test_IsContainDemistoWordValidator_is_invalid():
     results = IsContainDemistoWordValidator().is_valid(content_items)
     assert len(results) == 1
     assert results[0].message == expected_msg
+
+
+def test_ImagePathIntegrationValidator_is_valid_valid_case():
+    """
+    Given
+    content_items.
+    - Pack with valid readme and valid description contain only relative paths.
+    When
+    - Calling the ImagePathIntegrationValidator is_valid function.
+    Then
+    - Make sure that the pack isn't failing.
+    """
+    content_items = [
+        create_integration_object(
+            readme_content="![Example Image](../doc_files/image.png)",
+            description_content="valid description ![Example Image](../doc_files/image.png)",
+        ),
+    ]
+    assert not ImagePathIntegrationValidator().is_valid(content_items)
+
+
+def test_ImagePathIntegrationValidator_is_valid_invalid_case():
+    """
+        Given
+        content_items.
+        - Pack with:
+            1. invalid readme that contains absolute path.
+            2. description contains relative path that saved not under dec_files.
+    demisto_sdk/commands/validate/sdk_validation_config.toml
+
+        When
+        - Calling the ImagePathIntegrationValidator is_valid function.
+        Then
+        - Make sure that the pack is failing.
+    """
+    content_items = [
+        create_integration_object(
+            readme_content=" Readme contains absolute path:\n 'Here is an image:\n"
+            " ![Example Image](https://www.example.com/images/example_image.jpg)",
+            description_content="valid description ![Example Image](../../content/image.jpg)",
+        ),
+    ]
+    expected = (
+        "Invalid image path(s) detected. Please use relative paths instead in the following links:"
+        "\nhttps://www.example.com/images/example_image.jpgRelative image paths found outside the pack's "
```

Comment:
https://www.example.com/images/example_image.jpgRelative lets change to  have a space between the .jpg and the Relative.